release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1703,7 +1688,7 @@ In the following tables, features are marked with the following statuses:
 |General Availability
 
 |OpenStack Manila support for CSI resize
-|Not Available
+|General Availability
```

Comment:
The [4.18 release notes](https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/release_notes/ocp-4-18-release-notes#ocp-4-18-technology-preview-tables_release-notes) state that this feature is Not Available in 4.17.

Reply:
Well caught.

```suggestion
|Not Available
```